### PR TITLE
[TF 2.0 API Docs] Added Documentation to tf.keras.activations.elu

### LINKED ISSUE #27652: [TF 2.0 API Docs] tf.keras.activations.elu
**System information**
- TensorFlow version: 2.0 alpha
- Doc Link:
https://www.tensorflow.org/versions/r2.0/api_docs/python/tf/keras/activations/elu
https://github.com/tensorflow/tensorflow/blob/master/tensorflow/python/keras/activations.py

**- Links**
Links exist. Should format the link to the original paper (delete "-"). Also, should add authors and year of publishing 

E.g. 
```
Reference: "Fast and Accurate Deep Network Learning by Exponential Linear Units (ELUs)" (Clevert et al, 2015)
```

**- Definition**
Since it just says: `Exponential linear unit` we can modify it to the following - similar to the [original paper](https://arxiv.org/abs/1511.07289) :
```
The exponential linear unit (ELU) with `alpha` > 0 is:
`x` if `x > 0` and `alpha * (exp(x)-1)` if `x < 0`
The ELU hyperparameter `alpha` (α) controls the value to which an ELU saturates for negative net inputs.
ELUs diminish the vanishing gradient effect.
```
Followed by word-for-word stuff from the [original paper](https://arxiv.org/abs/1511.07289):
```
ELUs have negative values which pushes the mean of the activations closer to zero. 
Mean activations that are closer to zero enable faster learning as they bring the gradient closer to the natural gradient. 
ELUs saturate to a negative value when the argument gets smaller. 
Saturation means a small derivative which decreases the variation and the information that is propagated to the next layer."
```

**- Examples**
No examples given. Can add a modified example from the Intro to CNNs tutorial (where `elu` replaces ReLU - `relu`) 

E.g.
```
model = models.Sequential()
model.add(layers.Conv2D(32, (3, 3), activation='elu', input_shape=(28, 28, 1)))
model.add(layers.MaxPooling2D((2, 2)))
model.add(layers.Conv2D(64, (3, 3), activation='elu'))
model.add(layers.MaxPooling2D((2, 2)))
model.add(layers.Conv2D(64, (3, 3), activation='elu'))
```

**- Parameters**
Both params defined already but to aid the user it should state that `alpha` (α) should be set at 1.0  _by default_ and that:
```
`alpha` controls the value to which an ELU saturates for negative net inputs.
```
(source: [paper](https://arxiv.org/abs/1511.07289)) 

... instead of just 
```
`alpha`: A scalar, slope of negative section
```

**- Returns**
Defined but for clarity should say: 
```
The exponential linear unit (ELU) activation function: `x` if `x > 0` and `alpha * (exp(x) - 1)` if `x < 0`
```
 instead of simply `The exponential linear activation:...[equation]`

**- Raises**
Not defined.

**- Visuals**
Should be added to help the user. Example - see p.5 of the original paper: https://arxiv.org/pdf/1511.07289.pdf.

**We welcome contributions by users. Will you be able to update submit a PR (use the [doc style guide](https://www.tensorflow.org/community/documentation)) to fix the doc Issue?**
Yes

### PR DESCRIPTION
**Pull request by bharatr21**
Fix #27652 by adding required information to tf.keras.activations.elu [here](https://github.com/tensorflow/tensorflow/blob/master/tensorflow/python/keras/activations.py) including an image for visuals and example usage from the Intro to CNNs tutorial as suggested.

### CODE REVIEW

File: tensorflow/python/keras/activations.py [hunk 1]
```diff
@@ -68,19 +68,50 @@ def softmax(x, axis=-1):
 
 @keras_export('keras.activations.elu')
 def elu(x, alpha=1.0):
-  """Exponential linear unit.
+  """
+  The exponential linear unit (ELU) with `alpha` > 0 is:
+  `x` if `x > 0` and `alpha * (exp(x)-1)` if `x < 0`
+  The ELU hyperparameter `alpha` (α) controls the value to which an
+  ELU saturates for negative net inputs.ELUs diminish the
```
Comment (kyscg) [inline tensorflow/python/keras/activations.py]:
```suggestion
  ELU saturates for negative net inputs. ELUs diminish the
```

File: tensorflow/python/keras/activations.py [hunk 2]
```diff
@@ -68,19 +68,50 @@ def softmax(x, axis=-1):
...
-      The exponential linear activation: `x` if `x > 0` and
-        `alpha * (exp(x)-1)` if `x < 0`.
+      The exponential linear unit (ELU) activation function: `x` if `x > 0` and
+      `alpha * (exp(x) - 1)` if `x < 0`.
+
 
   Reference:
-      - [Fast and Accurate Deep Network Learning by Exponential
-        Linear Units (ELUs)](https://arxiv.org/abs/1511.07289)
+  [Fast and Accurate Deep Network Learning by Exponential Linear Units (ELUs)
```
Comment (kyscg) [inline tensorflow/python/keras/activations.py]:
```suggestion
      [Fast and Accurate Deep Network Learning by Exponential Linear Units (ELUs)
```

File: tensorflow/python/keras/activations.py [hunk 3]
```diff
@@ -68,19 +68,50 @@ def softmax(x, axis=-1):
...
+  (Clevert et al, 2015)](https://arxiv.org/abs/1511.07289)
```
Comment (kyscg) [inline tensorflow/python/keras/activations.py]:
```suggestion
      (Clevert et al, 2015)](https://arxiv.org/abs/1511.07289)
```

File: tensorflow/python/keras/activations.py [hunk 4]
```diff
@@ -68,19 +68,50 @@ def softmax(x, axis=-1):
 
 @keras_export('keras.activations.elu')
 def elu(x, alpha=1.0):
-  """Exponential linear unit.
+  """
```
Comment (kyscg) [inline tensorflow/python/keras/activations.py]: Maybe retain the title here?

**Review by kyscg — CHANGES_REQUESTED** [review]
Great contribution, could you make at the following changes??

They're mostly related to formatting, I've made them, you just have to accept them.

Cheers!!!

**Comment by kyscg** [comment]
We've signed them. @rthadur, please set CLA's to yes and review

Thanks!!

**Review by kyscg — DISMISSED** [review]

**Comment by Bharat123rox** [comment]
Sorry @kyscg, I realized I made a typo while rendering the image, which has now been corrected. Please re-review, apologies!!

**Comment by kyscg** [comment]
I was wondering about that, I should have mentioned it earlier

Thanks a lot for the change @Bharat123rox

File: tensorflow/python/keras/activations.py [hunk 5]
```diff
@@ -68,19 +68,50 @@ def softmax(x, axis=-1):
...
+
+  ELUs have negative values which pushes the mean of the activations
+  closer to zero.
+  Mean activations that are closer to zero enable faster learning as they
+  bring the gradient closer to the natural gradient.
+  ELUs saturate to a negative value when the argument gets smaller.
+  Saturation means a small derivative which decreases the variation
+  and the information that is propagated to the next layer.
+
+  ![](https://akiraaptx.files.wordpress.com/2017/01/screen-shot-2017-01-14-at-1-39-39-pm.png)
```
Comment (kyscg) [inline tensorflow/python/keras/activations.py]: Very informative graphic, good work @Bharat123rox

**Review by kyscg — DISMISSED** [review]

File: tensorflow/python/keras/activations.py (hunk 5 above)
Comment (Bharat123rox) [reply tensorflow/python/keras/activations.py]: I've done something similar for SELU in #27837, if you could review that for formatting as well...thanks!!

**Comment by kyscg** [comment]
@rthadur, gentle ping to review and set CLA's to yes

**Comment by rthadur** [comment]
@Bharat123rox thank you for your contribution , please make sure all the contributors have signed CLA

**Comment by Bharat123rox** [comment]
Yes, all contributors have signed the CLA @Bharat123rox and @kyscg and are
willing to merge contributions in source code

**Comment by kyscg** [comment]
@rthadur @akshaym Gentle ping to review

File: tensorflow/python/keras/activations.py [hunk 6]
```diff
@@ -68,19 +68,50 @@ def softmax(x, axis=-1):
 
 @keras_export('keras.activations.elu')
 def elu(x, alpha=1.0):
-  """Exponential linear unit.
+  """Exponential Linear Unit.
```
Comment (fchollet) [inline tensorflow/python/keras/activations.py]: Need a blank line below this line.

File: tensorflow/python/keras/activations.py [hunk 7]
```diff
@@ -68,19 +68,50 @@ def softmax(x, axis=-1):
...
+  The exponential linear unit (ELU) with `alpha` > 0 is:
```
Comment (fchollet) [inline tensorflow/python/keras/activations.py]: `alpha > 0`

File: tensorflow/python/keras/activations.py [hunk 8]
```diff
@@ -68,19 +68,50 @@ def softmax(x, axis=-1):
...
+  `x` if `x > 0` and `alpha * (exp(x)-1)` if `x < 0`
```
Comment (fchollet) [inline tensorflow/python/keras/activations.py]: This is hard to read; format it as a 2-point list.

File: tensorflow/python/keras/activations.py [hunk 9]
```diff
@@ -68,19 +68,50 @@ def softmax(x, axis=-1):
...
+  The ELU hyperparameter `alpha` (α) controls the value to which an
```
Comment (fchollet) [inline tensorflow/python/keras/activations.py]: No need for the greek letter.

File: tensorflow/python/keras/activations.py (hunk 5 above)
Comment (fchollet) [reply tensorflow/python/keras/activations.py]: Because this image is hosted by a third-party, we cannot it include it in the docstring. It would need to be hosted on tensorflow.org.

**Review by fchollet — CHANGES_REQUESTED** [review]
Thank you for the PR.

**Comment by Bharat123rox** [comment]
@rthadur or @gbaned please add the `cla:yes` label, @kyscg and/or @fchollet please re-review

**Comment by mellanox-github** [comment]
Can one of the admins verify this patch?

File: tensorflow/python/keras/activations.py [hunk 10]
```diff
@@ -68,19 +68,48 @@ def softmax(x, axis=-1):
...
+  Mean activations that are closer to zero enable faster learning as they
+  bring the gradient closer to the natural gradient.
+  ELUs saturate to a negative value when the argument gets smaller.
+  Saturation means a small derivative which decreases the variation
+  and the information that is propagated to the next layer.
+
+  Example Usage:
+
+  ```python3
+  model = models.Sequential()
```
Comment (lamberta) [inline tensorflow/python/keras/activations.py]: Can this snippet be changed to DocTest format so it can be run and tested? See https://www.tensorflow.org/community/contribute/docs_ref

**Comment by gbaned** [comment]
@Bharat123rox Can you please resolve conflicts? Thanks!

**Comment by Bharat123rox** [comment]
@gbaned Conflict is resolved, thanks!

**Review by lamberta — DISMISSED** [review]

**Comment by gbaned** [comment]
@Bharat123rox Can you please address Ubuntu Sanity errors? Thanks!

**Comment by Bharat123rox** [comment]
@gbaned The Ubuntu Sanity errors should be fixed, thanks!

**Comment by gbaned** [comment]
@lamberta Can you please review this PR ? Thanks!

**Review by lamberta — APPROVED** [review]
